__mock__/process-data.js

```diff
@@ -0,0 +1,3 @@
+const processData = data => data;
+
+module.exports = processData;
```

Comment:
I imagine this is here because you copy/pasted the other project. You should get rid of it, though, because it isn't actually doing anything.